website/docs/components/code-block/partials/code/component-api.md

```diff
@@ -0,0 +1,41 @@
+## Component API
+
+This component uses [prism.js](https://prismjs.com/) under the hood.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="value" @type="string">
+    The text/code content for the CodeBlock.
```

Comment:
We should specify how this should be provided. Eg. should already be encoded (eg. for HTML). What other things the consumer should know about this important argument?

Reply:
Tried to clarify this in [4557418](https://github.com/hashicorp/design-system/pull/1749/commits/455741830da8a3954eadd3d711c27c9bad6d09d5)